packages/react/src/components/accordion/accordion-styles.ts

```diff
@@ -0,0 +1,51 @@
+import styled, { css } from 'styled-components';
+import { Theme } from '../../themes';
+
+export const StyledAccordionGroup = styled.div`
+    display: flex;
+    flex-direction: column;
+`;
+
+export const AccordionSectionStyled = (isExpanded: boolean, theme: Theme) => css`
+    background: ${theme.greys['colored-white']};
+    border: ${isExpanded ? `1px solid ${theme.greys['dark-grey']}` : '0'};
+    border-radius: 0 0 4px 4px;
+    border-top-width: 0;
+    color: #1b1c1e;
+    font-size: 0.75rem;
+    font-weight: 400;
+    height: ${isExpanded ? 'auto' : '0'};
+    letter-spacing: 0.24px;
+    line-height: 1.7;
+    margin-bottom: 8px;
+    min-height: 0;
+    overflow: hidden;
+    padding: ${isExpanded ? '16px 24px 24px 40px' : '0'};
+    transition: height 300ms cubic-bezier(0.25, 0.46, 0.45, 0.94) 0ms;
+` as any;
+
+export const HeadingStyled = (isExpanded: boolean, theme: Theme) => css`
+    border: 1px solid ${theme.greys['dark-grey']};
+    border-bottom-color: ${isExpanded ? theme.greys.grey : theme.greys['dark-grey']};
+    border-radius: ${isExpanded ? '4px 4px 0px 0px' : '4px'};
+` as any;
+
+export const ButtonStyled = (isExpanded: boolean, theme: Theme) => css`
```

Comment:
When the panel is disabled, the background-color should be `#F1F2F2`, The Icon: `#B7BBC2` & The border: `#DBDEE1`
Storybook:
<img width="165" alt="Screenshot 2023-08-21 at 1 05 27 PM" src="https://github.com/kronostechnologies/design-elements/assets/3722754/fcfec1c3-ef7e-48f9-bd79-438c84564a45">
Figma:
<img width="411" alt="Screenshot 2023-08-21 at 1 05 53 PM" src="https://github.com/kronostechnologies/design-elements/assets/3722754/44296d19-e08c-4328-98d0-56a54f7fb3d8">

Reply:
done